en/html/README.md

```diff
@@ -201,12 +201,12 @@ $ git push
 
 {% filename %}PythonAnywhere command-line{% endfilename %}
 ```
-$ cd ~/<your-pythonanywhere-username>.pythonanywhere.com
+$ cd ~/<your-pythonanywhere-domain>.pythonanywhere.com
 $ git pull
 [...]
 ```
 
-(Remember to substitute `<your-pythonanywhere-username>` with your actual PythonAnywhere username, without the angle-brackets).
+Remember to substitute `<your-pythonanywhere-domain>` with your actual PythonAnywhere domain name, without the angle-brackets. Your domain name is normally your PythonAnywhere user name, but in some cases it might be a bit different (such as if your user name contains capital letters). So if this command doesn't work, use the `ls` (list files) command to find your actual domain/folder name, and then `cd` to there.
```

Comment:
In the order of the chapter in the tutorial, this (and not `en/django_templates/README.md`) seems indeed to be the first occurrence, so it makes sense to have the extended explanation here, as you suggest.

Though:
- Didn't you suggest to have the copy-pastable `$USER` solution at that first occurrence? (I'm fine with dropping that in favor of consistency. Just want to make sure whether it's a conscious decision.)
- Why "Remember to" if this substitution didn't occur in the tutorial before this point?

Reply:
Yeah, I thought it was better to be consistent. You're right about "remember to". That was in this page before and I left it, but you're right it should be taken out of this one.

Reply:
Also, just to point out: the $USER was not actually in the first occurrence. It is in en/django_templates/README.md -- so I made that spot the same syntax as all the others already were (including this first occurrence, which is in the en/html file).

Reply:
> Also, just to point out: the $USER was not actually in the first occurrence. It is in en/django_templates/README.md -- [...]

Yeah, that was what I was trying to point out. :+1: